src/main/java/com/datapipe/jenkins/vault/buildstep/VaultReadStep.java

```diff
@@ -0,0 +1,96 @@
+package com.datapipe.jenkins.vault.buildstep;
+
+import static com.datapipe.jenkins.vault.configuration.VaultConfiguration.DescriptorImpl.DEFAULT_ENGINE_VERSION;
```

Comment:
The project has Intellij Code styles and checkstyle for imports.
A-Z and static last and no star imports.